doc/introduction/chemistry.rst

```diff
@@ -0,0 +1,192 @@
+        name='h2',
+        geo_file='h2.xyz',
+        charge=0,
+        multiplicity=1,
+        basis_set='sto-3g',
+        n_active_electrons=2,
+        n_active_orbitals=2
+    )
+
+where:
+
+* ``h`` is the qubit Hamiltonian of the molecule represented as an
+  `OpenFermion <https://github.com/quantumlib/OpenFermion>`_ ``QubitOperator``, and
+
+* ``nr_qubits`` is the number of qubits  needed to represent it.
+
+Internally, :func:`~.generate_mol_hamiltonian` calls the following functions in order
+to generate the qubit Hamiltonian:
+
+.. currentmodule:: pennylane_qchem.qchem
+
+.. autosummary::
+
+    read_structure
+    gen_meanfield_data
+    gen_active_space
+    gen_hamiltonian_pauli_basis
+
+
+For more fine-grained control, these functions may be
+called independently as required to obtain the
+electronic Hamiltonian of the molecule represented in the basis of Pauli matrices.
+
+
+Importing molecular structure data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The atomic structure of a molecule can be imported from an external file using the
+:func:`~.read_structure` function:
+
+.. code-block:: python
+
+    >>> geometry = qml.qchem.read_structure('h2o.SDF')
+    >>> print(geometry)
+    [['H', (-0.0211, -0.002, 0.0)], ['O', (0.8345, 0.4519, 0.0)], ['H', (1.4769, -0.273, 0.0)]]
+
+The geometry of the molecule is returned as a list containing the symbol and the Cartesian
+coordinates of each atomic species.
+
+.. note::
+
+    The `xyz <https://en.wikipedia.org/wiki/XYZ_file_format>`_ format is supported
+    out of the box. Additionally, if `Open Babel <https://openbabel.org/>`_ is
+    installed, `any format recognized by Open Babel
+    <https://openbabel.org/wiki/Category:Formats>`_ is also supported.
+
+    See the :func:`~.read_structure` function for more details.
+
+Solving the Hartree-Fock equations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :func:`~.gen_meanfield_data` function utilizes  `OpenFermion-PySCF <https://github
+.com/quantumlib/OpenFermion-PySCF>`_ and `OpenFermion-Psi4 <https://github
+.com/quantumlib/OpenFermion-Psi4>`_ plugins to solve the Hartree-Fock equations for the molecule
+using the electronic structure packages `PySCF <https://github.com/sunqm/pyscf>`_ and `Psi4
+<http://psicode.org/>`_, respectively.
+
+For this, it is required to specify a string to label the molecule, its net charge, the `spin
+multiplicity <https://en.wikipedia.org/wiki/Multiplicity_(chemistry)>`_ and the  `atomic basis
+functions <https://www.basissetexchange.org/>`_.
+
+.. code-block:: python
+
+    geometry = qml.qchem.read_structure('h2o.SDF')
+    hf_data = qml.qchem.gen_meanfield_data(
+        'water',
+        geometry,
+        charge=0,
+        multiplicity=1,
+        basis_set='sto-3g',
+        qc_package='pyscf'
+    )
+
+The output variable ``hf_data`` stores the path to the directory containing the file ``'water
+.hdf5'`` with the Hartree-Fock electronic structure of the water molecule.

```

Comment:
are newlines rendered to spaces in the docs?

Reply:
Good catch!
```suggestion
with the Hartree-Fock electronic structure of the water molecule.
```